src/test/java/com/puppycrawl/tools/checkstyle/api/AutomaticBeanTest.java

```diff
@@ -193,14 +225,28 @@ public void testBeanConverters() throws Exception {
         config.addAttribute("accessModifiers", "public, private");
         bean.configure(config);
 
-        assertArrayEquals(new String[] {"a", "b", "c"}, bean.strings, "invalid result");
-        assertEquals(".*", bean.pattern.pattern(), "invalid result");
-        assertEquals(SeverityLevel.ERROR, bean.severityLevel, "invalid result");
-        assertEquals(Scope.PUBLIC, bean.scope, "invalid result");
-        assertEquals(new URI("http://github.com"), bean.uri, "invalid result");
-        assertArrayEquals(new AccessModifierOption[] {AccessModifierOption.PUBLIC,
-            AccessModifierOption.PRIVATE}, bean.accessModifiers,
-                "invalid result");
+        final String message = "invalid result";
+        assertWithMessage(message)
+                .that(bean.strings)
+                .isEqualTo(new String[] {"a", "b", "c"});
```

Comment:
Please avoid using `isEqualTo` for arrays and collections. Suggested change
```
        assertWithMessage(message)
                .that(bean.strings).asList()
                .containsExactly("a", "b", "c");
``` 
It produces more readable violation message.

Reply:
done